### ISSUE TITLE
OOMMF SIMULATIONS

### ISSUE THREAD

**Issue by shivanirstogi** [davidcortesortuno/oommf_sk_number#2]
![Error](https://user-images.githubusercontent.com/85611477/126881213-14ec8b22-de59-4927-9e9b-eaf235ff4a37.png)
@davidcortesortuno I am using your library for computing skyrmion numbers. Your library helped me a lot in understanding things easily. Now, I am assigned to get the topological charge density and its plot. Being new to this field and thing, I am unable to do some things on my own. I tried but was unable to grasp that one.  

Can you please help me understand the topological charge density and to get its plot and values. I tried your library but I am getting an error message.

Kindly help me out, if possible. This thing can help me in a great way to solve my problems.

Thanks and Regards,
Shivani Rastogi

**Comment by shivanirstogi** [comment]
I am also attaching the OVF file on which I am looking for results. Please help 
 me out boss. 
[https://drive.google.com/file/d/1rxbr1MRfpvAL7HOeGZR8sXChKBodxrlB/view?usp=sharing](url)

**Comment by davidcortesortuno** [comment]
Hi, this library is no longer developed, as mentioned in the README:
```
This tool is not developed anymore and the project has been migrated to OOMMFPy:
```
https://github.com/davidcortesortuno/oommfpy

Have you tried OOMMFPy ?

**Comment by shivanirstogi** [comment]
Yes sir, already installed OOMMFpy. From tools section, I used Plot file to get charge density plot. I tried but unable to parse files in the py file, its showing syntax errors.

**Comment by davidcortesortuno** [comment]
what kind of errors exactly?
